filters/general/filter_53_AWAvenueAdsRule/metadata.json

```
@@ -0,0 +1,14 @@
+{
+    "filterId": "AWAvenue-Ads-Rule",
+    "id": 53,
+    "name": "秋风广告规则",
+    "description": "Use Adblock syntax to fight against various advertising SDKs in Android applications from the network level , prevent them from loading.",
+    "timeAdded": 1699370433972,
+    "homepage": "https://awavenue.top/",
+    "expires": "15 days",
+    "displayNumber": 2,
+    "environment": "prod",

```

`dev` first. If this pr will be merged, we will create a new pr with `dev` => `pr`, and merge it after successful testing.
```suggestion
    "environment": "dev",

```